WalletWasabi/WabiSabi/Client/CoinJoinManager.cs

```diff
@@ -199,16 +199,11 @@ protected override async Task ExecuteAsync(CancellationToken stoppingToken)
 	/// <summary>
 	/// Mark all the outputs we had in any of our wallets used.
 	/// </summary>
-	private void MarkDestinationsUsed(CoinJoinTracker finishedCoinJoin)
+	private void MarkDestinationsUsed(Wallet wallet, ImmutableList<Script> outputs)
```

Comment:
The previous code looked at all the wallets instead of just one. This is important at wallet to wallet transfer.

Reply:
I didn't know it is possible to have the same script in two wallets. Anyway, I added cannot hurt.

Reply:
It is not possible to have the same script in two wallets. That isn't what this is about. It's about what if I send to another wallet in a coinjoin. In that case the script is only in the receiver wallet and not in the originating wallet.